docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -1906,7 +1906,9 @@ Returns all accounts owned by the provided program Pubkey
 - `memcmp: <object>` - compares a provided series of bytes with program account data at a particular offset. Fields:
 
   - `offset: <usize>` - offset into program account data to start comparison
-  - `bytes: <string>` - data to match, as base-58 encoded string and limited to less than 129 bytes
+  - `bytes: <string>` - data to match, as encoded string
+  - `encoding: <string>` - encoding for filter `bytes` data, either "base58" or "base64". Data is limited in size to less than 129 decoded bytes.
+    **NEW: This field, and base64 support generally, is only available in solana-core v1.10.30 or newer. Please omit when querying nodes on earlier versions**
```

Comment:
should we mention 1.11.2 here too? (eg. 1.10.30 < 1.11.1 < 1.11.2)

Reply:
Oh, oops. I did decide not to backport but missed fixing this version. Thanks for catching!